src/views/v2/commerce/checkout/cart/empty/description.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Tickets Commerce: Checkout with Empty Cart Description
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/cart/empty/description.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this                  [Global] Template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var array[]          $items                 [Global] List of Items on the cart to be checked out.
+ * @var string           $paypal_attribution_id [Global] What is our PayPal Attribution ID.
+ * @var bool             $must_login            [Global] Whether login is required to buy tickets or not.
+ * @var string           $login_url             [Global] The site's login URL.
+ * @var string           $registration_url      [Global] The site's registration URL.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ * @var int              $section               Which Section that we are going to render for this table.
+ * @var \WP_Post         $post                  Which Section that we are going to render for this table.
+ */
+
+if ( ! empty( $items ) ) {
+	return;
+}
+
+if ( $is_tec_active ) {
+	$description = sprintf(
+		// Translators: %1$s: Opening `<a>` tag. %2$s: Plural `events` in lowercase. %3$s: Closing `</a>` tag. %4$s: Plural `tickets` in lowercase.
+		__( 'Please %1$sbrowse %2$s%3$s and add %4$s to check out.', 'event-tickets' ),
```

Comment:
```suggestion
		__( 'Please %1$sbrowse %2$s%3$s and add %4$s to checkout.', 'event-tickets' ),
```

Reply:
Nice! that's coming from design https://www.figma.com/file/TvIXoUEJAGb0AptZI260UO/TicketsCommerce?node-id=874%3A116 - Have to confirm with David. I agree that the correct term for e-Commerce is checkout.

Reply:
@rafsuntaskin David's reply here: https://cloudup.com/cHycAYgpRjT

We're staying with `check out` for now, due to his explanation.